doc/introduction/interfaces/numpy.rst

```diff
@@ -365,7 +370,7 @@ to the ``scipy.minimize`` function:
     def cost(x):
         return np.abs(circuit(x) - 0.5) ** 2
 
-    params = np.array([0.011, 0.012, 0.05])
+    params = np.array([0.011, 0.012, 0.05], requires_grad=True)
```

Comment:
Did the output after line 381 not change?

Reply:
OK.... 
```
>>> minimize(cost, params, method='BFGS', jac=qml.grad(cost))
```

is raising a warning... How would we fix that?

Reply:
fixed.